src/main/frontend/src/routes/aggregates.svelte

```diff
@@ -2,34 +2,31 @@
 	import CardForm from "../components/CardForm.svelte";
 	import AggregateCard from "../components/aggregates/AggregateCard.svelte";
 	import AggregateDialog from "../components/aggregates/AggregateDialog.svelte";
-	import { aggregateSettings, setLocalStorage } from "../stores";
+	import { aggregateSettings, getLocalStorage, setLocalStorage } from "../stores";
 	import { SlideGroup, SlideItem, Icon } from "svelte-materialify/src";
 	import { mdiPlus } from "@mdi/js";
-import Button from "svelte-materialify/src/components/Button";
+	import Button from "svelte-materialify/src/components/Button";
 
-	let dialogActive = false;
-	let editMode = false;
+	let dialogActive = getLocalStorage("aggregateDialogState") ? getLocalStorage("aggregateDialogState").dialogActive : false;
+	let editMode = getLocalStorage("aggregateDialogState") ? getLocalStorage("aggregateDialogState").editMode : false;
 
-	let currentId;
+	let oldAggregate = getLocalStorage("aggregateDialogState") ? getLocalStorage("aggregateDialogState").oldAggregate : undefined;
```

Comment:
naming is much better: `oldAggregate` is only used in editMode, inside the AggregateDialog we use it to get our initial state. Inside, `newAggregate` is used to save the new/changed data.